src/Commands/core/UpdateDBCommands.php

```diff
@@ -205,14 +205,14 @@ public static function updateDoOne($module, $number, $dependency_map, &$context)
                 Drush::logger()->notice("Update started: $function");
                 $ret['results']['query'] = $function($context['sandbox']);
                 $ret['results']['success'] = true;
-            } catch (\Throwable $e) {
-                // PHP 7 introduces Throwable, which covers both Error and Exception throwables.
-                $ret['#abort'] = ['success' => false, 'query' => $e->getMessage()];
-                Drush::logger()->error($e->getMessage());
             } catch (\Exception $e) {
                 // In order to be compatible with PHP 5 we also catch regular Exceptions.
                 $ret['#abort'] = ['success' => false, 'query' => $e->getMessage()];
                 Drush::logger()->error($e->getMessage());
+            } catch (\Throwable $e) {
```

Comment:
I got the current ordering from https://github.com/drupal/drupal/blob/8.6.x/core/includes/bootstrap.inc#L597-L613. Happy to go with your ordering if thats better.

Reply:
Interesting. Maybe I need another test.  😛

Reply:
Yeah, sure enough, the Drupal-inspired order works. Would be pretty surprising / impossible for it to be otherwise.